Classes/Resource/OnlineMedia/Helpers/AbstractSrgssrHelper.php

```diff
@@ -19,42 +9,22 @@
 
 abstract class AbstractSrgssrHelper extends AbstractOnlineMediaHelper
 {
-    /**
-     * @var string
-     */
-    protected $channelName = '';
+    protected string $channelName = '';
 
-    /**
-     * @var string
-     */
-    protected $siteHostname = '';
+    protected string $siteHostname = '';
 
-    /**
-     * @var $extension = '';
-     */
     protected $extension = '';
```

Comment:
Didn't phpstan complain here? ... maybe:
```suggestion
    protected string $extension = '';
```